tests/microbenchmarks/all_reduce.py

```
@@ -49,80 +49,120 @@ def allreduce_benchmark(dtype: str,
+                        start[i].elapsed_time(stop[i])
+                        for i in range(outer_loop)
+                    ]
+                    median_ms = sorted(runtimes)[len(runtimes) // 2]
+
+                    if fusion == AllReduceFusionOp.NONE:
+                        allreduce_ref = (input * world_size)**inner_loop
+                        torch.testing.assert_close(output, allreduce_ref)
+
+                    if mapping.rank == 0:
+                        print(
+                            f"{mapping.world_size:<15}, {dtype:<10}, {size:<15}, {strategy.name:<10}, {fusion.name:<20}, {version:<10}, {median_ms:<10.2f}"
```

Ruff (0.12.2)

Line too long (155 > 120)

(E501)